code/controllers/subsystem/minor_mapping.dm

```diff
@@ -60,10 +60,11 @@ SUBSYSTEM_DEF(minor_mapping)
 
 /proc/find_satchel_suitable_turfs()
 	var/list/suitable = list()
+	var/list/unsuitable = list(/turf/open/floor/catwalk_floor, /turf/open/floor/glass)
 
 	for(var/z in SSmapping.levels_by_trait(ZTRAIT_STATION))
 		for(var/t in block(locate(1,1,z), locate(world.maxx,world.maxy,z)))
-			if(isfloorturf(t) && !isplatingturf(t))
+			if((isfloorturf(t) && !isplatingturf(t)) && !(t in unsuitable))
```

Comment:
`t` is an instance of a turf, and `unsuitable` is a list of types, you're checking for an instance of a turf in a list of types